src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+
+import { FormType, TodoType } from '../../types';
+import usersFromServer from '../../api/users';
+import { findMaxId } from '../../helpers/findMaxTodoId';
+
+type Props = {
+  addTodo: (todo: TodoType) => void,
+  todos: TodoType[]
+};
+
+const initialFormValue = {
+  titleInput: '',
+  userSelect: 0,
+};
+
+export const TodoForm: React.FC<Props> = ({ addTodo, todos }) => {
+  const [formValues, setFormValues] = useState<FormType>(initialFormValue);
+  const [inputError, setInputError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+
+  const handleSetFormValues
+  = (event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+    let value: number | string = '';
+
+    if (event.target.name === 'userSelect') {
+      value = Number(event.target.value);
+      setSelectError(false);
+    }
+
+    if (event.target.name === 'titleInput') {
+      value = event.target.value.trimStart();
+      setInputError(false);
+    }
+
+    setFormValues(prevState => ({
+      ...prevState,
+      [event.target.name]: value,
+    }));
+  };
+
+  const handleAddTodo = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+
+    const { titleInput: title, userSelect: userId } = formValues;
+
+    if (!title) {
+      setInputError(true);
+    }
```

Comment:
setInputError(!title), same with select

Reply:
I don't agree with you. My main argument is that there are 2 separate errors for two different types of inputs and therefore there should be two stakes. If I combine the errors into one, it will be work not properly